src/NzbDrone.Api/Commands/CommandResource.cs

```diff
@@ -8,11 +8,73 @@ public class CommandResource : RestResource
     {
         public String Name { get; set; }
         public String Message { get; set; }
-        public DateTime StartedOn { get; set; }
-        public DateTime StateChangeTime { get; set; }
-        public Boolean SendUpdatesToClient { get; set; }
-        public CommandStatus State { get; set; }
+        
+        public Command Body { get; set; }
+        public CommandPriority Priority { get; set; }
+        public CommandStatus Status { get; set; }
+        public DateTime Queued { get; set; }
+        public DateTime? Started { get; set; }
+        public DateTime? Ended { get; set; }
+        public TimeSpan? Duration { get; set; }
+        public string Exception { get; set; }
+        public CommandTrigger Trigger { get; set; }
+
+        //Legacy
+        public CommandStatus State
```

Comment:
We should set up integration tests to check the api for known third-party usage.